database/migrations/2024_07_26_143301_add_checkout_for_all_types_to_accessories.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use App\Models\Setting;
+use App\Models\User;
+use Illuminate\Support\Facades\DB;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        if (Schema::hasTable('accessories_users')) {
+            Schema::rename('accessories_users', 'accessories_checkout');
+
+            Schema::table('accessories_checkout', function (Blueprint $table) {
+                if (!Schema::hasColumn('accessories_checkout', 'assigned_type')) {
+                    $table->string('assigned_type')->nullable();
+                }
+            });
+        }
+        
+        DB::update('update '.DB::getTablePrefix().'accessories_checkout set assigned_type = \'App\\\\Models\\\\User\' where assigned_type is null', []);
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        if (Schema::hasTable('accessories_checkout')) {
+            Schema::rename('accessories_checkout', 'accessories_users');
```

Comment:
We probably want to drop the column created in the up migration as well, no?

Reply:
Yes, I will adjust that.

Reply:
Adjusted